lib/MHA/Config.pm

```diff
@@ -263,6 +263,12 @@ sub parse_server {
   $value{skip_init_ssh_check} = 0
     if ( !defined( $value{skip_init_ssh_check} ) );
 
+  $value{skip_version_check} = $param_arg->{skip_version_check};
+  $value{skip_version_check} = $default->{skip_version_check}
+    if ( !defined( $value{skip_version_check} ) );
```

Comment:
But this is a no values option, unlike the ssh port configuration. There is no `default` for it.
Shouldn't it be more correct to use the same pattern as `skip_init_ssh_check` in that case?
https://github.com/insom/mha4mysql-manager/blob/30f28857c09a7a8399ec054231c468e211327571/lib/MHA/Config.pm#L262-L264